src/main/scala/cromwell/engine/workflow/ExecutionStoreKey.scala

```diff
@@ -0,0 +1,66 @@
+package cromwell.engine.workflow
+
+import cromwell.binding._
+import cromwell.engine.ExecutionStatus
+import cromwell.engine.workflow.WorkflowActor.ExecutionStore
+
+sealed trait ExecutionStoreKey {
+  val scope: Scope
+
+  val index: Option[Int]
+
+  val parent: Option[ExecutionStoreKey]
+}
+
+
+case class CallKey(override val scope: Call,
```

Comment:
yeah i don't care enough to ding a PR but I don't want it to really become A Thing. I realize my lack of self consistency on this one but it just seems kinda awkward to me.